src/test/breakpoints/breakpointsTest.ts

```diff
@@ -618,4 +618,77 @@ describe('breakpoints', () => {
       );
     });
   });
+
+  describe('hit count', () => {
```

Comment:
My PR has tests for these too 🤷‍♂

Reply:
If we test it twice that means it's twice as reliable!

I'll merge my test with yours once both our PRs are in master.